UI/js-src/lsmb/DateTextBox.js

```diff
@@ -63,6 +63,7 @@ define([
                      which the widget expects */
             this.value = this.parse(this._formattedValue, this.constraints);
          }
+         /* eslint no-restricted-globals: 0 */
```

Comment:
Instead of suppressing it, shouldn't `isNan()` be replaced by `Number.isNan()`?

Reply:
Not on this one. Number.isNan converts input to number, then validates.